src/common/selectors.js

```diff
@@ -448,6 +448,16 @@ export const createGetTags = collectionSelectors => {
   return _extendCollectionSelector(getTags, 'tag')
 }
 
+export const getLastShutdownTime = create(
+  (_, vm) => vm.id,
```

Comment:
This selector should be configurable.

You should create a factory instead of a plain selector (`createGetVmShutdownTime`).